packages/web-components/src/tabs/tab-list/tab-list.ts

```diff
@@ -0,0 +1,82 @@
+import { attr, booleanConverter } from '@microsoft/fast-element';
+import { FASTTabs } from '@microsoft/fast-foundation';
+
+export interface TabData {
+  id: string;
+  x: number;
+  y: number;
+  height: number;
+  width: number;
+}
+/**
+ * TabList extends FASTTabs and is used for constructing a fluent-tab-list custom html element.
+ *
+ * @class TabList component
+ * @public
+ */
+export class TabList extends FASTTabs {
```

Comment:
So, I don't know that extending here with the name `TabList` makes sense given that this is the _whole_ thing. The tablist does not include tabpanels - this class from FAST maps explicitly to the APG `Tabs` Component. A tablist is constructed, but I don't think it's going to be clear or correct to convolute these two simply because the React implementation doesn't implement panels (from what I am grokking).

If we'd like to match the naming, you probably need to change the implementation.

Reply:
So what do you think? Rename to Tabs and talk about the differences here in the readme?

Reply:
If we want to keep the FAST Foundation code, I would rename to Tabs and document. Otherwise, we can implement according to the React implementation.